app/helpers/application_helper.rb

```diff
@@ -55,9 +55,11 @@ def conditional_text(variable, prefix = '', suffix = '')
     end
   end
 
+  # This was returing the name, but now returns the estate name -
+  # the top-level organising unit for a particular prison
   def prison_name_for_id(nomis_id)
```

Comment:
Rather than have a comment here shouldn't we alter the method name to reveal the thing its returning?